packages/gatsby-plugin-tinacms/README.md

```diff
@@ -36,20 +22,28 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-tinacms',
       options: {
-        plugins: [
-          //
-        ],
+        enabled: process.env.NODE_ENV !== 'production',
         sidebar: {
-          hidden: process.env.NODE_ENV === 'production',
           position: 'displace',
         },
+        plugins: [
+          //...
+        ],
       },
     },
   ],
 }
 ```
 
-## Next Steps
+## Options
+
+```ts
+export interface GatsbyPluginTinacmsOptions {
+  enabled?: boolean
+  sidebar: TinaCMSConfig['sidebar']
+  toolbar: TinaCMSConfig['toolbar']
```

Comment:
Maybe we could add a link to the sidebar/toolbar docs 

https://tinacms.org/docs/ui#enabling-the-user-interface